### PR TITLE
refactor(editor): Remove old execution view route and its dependants

### PR DESCRIPTION
**Pull request by cstuncsik**

### CODE REVIEW

File: packages/editor-ui/src/views/NodeView.vue [hunk 1]
```diff
@@ -460,7 +457,6 @@ export default mixins(
 				return CanvasHelpers.getBackgroundStyles(
 					this.nodeViewScale,
 					this.uiStore.nodeViewOffsetPosition,
-					this.isExecutionPreview,
```
Comment (MiloradFilipovic) [inline packages/editor-ui/src/views/NodeView.vue:463]: We still need this.

File: packages/editor-ui/src/views/NodeView.vue [hunk 2]
```diff
@@ -533,7 +529,6 @@ export default mixins(
 				dropPrevented: false,
 				renamingActive: false,
 				showStickyButton: false,
-				isExecutionPreview: false,
```
Comment (MiloradFilipovic) [inline packages/editor-ui/src/views/NodeView.vue:536]: And this.

File: packages/editor-ui/src/views/NodeView.vue [hunk 3]
```diff
@@ -679,70 +674,6 @@ export default mixins(
 				this.onToggleNodeCreator({ source, createNodeActive: true });
 				this.nodeCreatorStore.showTabs = false;
 			},
-			async openExecution(executionId: string) {
```
Comment (MiloradFilipovic) [inline packages/editor-ui/src/views/NodeView.vue:682]: And this.

File: packages/editor-ui/src/views/NodeView.vue [hunk 4]
```diff
@@ -3149,7 +3075,6 @@ export default mixins(
 					if (json && json.command === 'openWorkflow') {
 						try {
 							await this.importWorkflowExact(json);
-							this.isExecutionPreview = false;
```
Comment (MiloradFilipovic) [inline packages/editor-ui/src/views/NodeView.vue:3152]: This also.

File: packages/editor-ui/src/views/NodeView.vue [hunk 5]
```diff
@@ -3160,24 +3085,6 @@ export default mixins(
 								type: 'error',
 							});
 						}
-					} else if (json && json.command === 'openExecution') {
```
Comment (MiloradFilipovic) [inline packages/editor-ui/src/views/NodeView.vue:3163]: And this.

File: packages/editor-ui/src/views/canvasHelpers.ts [hunk 6]
```diff
@@ -505,19 +505,11 @@ export const getMidCanvasPosition = (scale: number, offset: XYPosition): XYPosit
 	return getRelativePosition(editorWidth / 2, (editorHeight - HEADER_HEIGHT) / 2, scale, offset);
 };
 
-export const getBackgroundStyles = (scale: number, offsetPosition: XYPosition, executionPreview: boolean) => {
```
Comment (MiloradFilipovic) [inline packages/editor-ui/src/views/canvasHelpers.ts:508]: And this.

File: packages/editor-ui/src/views/canvasHelpers.ts [hunk 7]
```diff
@@ -505,19 +505,11 @@ export const getMidCanvasPosition = (scale: number, offset: XYPosition): XYPosit
...
+export const getBackgroundStyles = (scale: number, offsetPosition: XYPosition) => {
 	const squareSize = GRID_SIZE * scale;
 	const dotSize = 1 * scale;
 	const dotPosition = (GRID_SIZE / 2) * scale;
 
-	if (executionPreview) {
```
Comment (MiloradFilipovic) [inline packages/editor-ui/src/views/canvasHelpers.ts:513]: And also this.

**Review by MiloradFilipovic — CHANGES_REQUESTED** [review]
Nice job on this. It has been needing a cleanup.
However, you've cut a bit too much with this one: Execution previews are not loading anymore and execution preview canvas doesn't have it's distinct striped background (compared to main node view canvas which have dots).
I have left comments in PR files which parts I think need to get back, but please test and make sure we can still see execution previews after refactor.

**Comment by MiloradFilipovic** [comment]
A bit of context that may be helpful:
We are still using `NodeView` to display executions preview, it's just that it's used inside an `iframe` on _Executions_ tab. So [`openExecution`](https://github.com/n8n-io/n8n/blob/c2da21621fb08c676bd3dc5e2a9de613a4f135f3/packages/editor-ui/src/views/NodeView.vue#L682) function is still needed, but it's only used [here](https://github.com/n8n-io/n8n/blob/c2da21621fb08c676bd3dc5e2a9de613a4f135f3/packages/editor-ui/src/views/NodeView.vue#L3163), so all of [this logic](https://github.com/n8n-io/n8n/blob/c2da21621fb08c676bd3dc5e2a9de613a4f135f3/packages/editor-ui/src/views/NodeView.vue#L2182) in `initView` can go away (which was already done in this PR 🙌). Another thing is the `isExecutionPreview` data member which is used to distinguish `NodeView` used to display workflows from the ones used to preview executions, for now we use it only to set canvas background but it's important none the less.

For reference, this is how canvas should look like:

SCR-20221119-fmm

And this is what I see on this branch

SCR-20221119-fnv


And this is what I see on this branch:

**Review by MiloradFilipovic — APPROVED** [review]
Looks good to me.

**Comment by janober** [comment]
Got released with `n8n@0.211.1`

**Pull request merged**